lsp/nls/Cargo.toml

```diff
@@ -14,10 +14,6 @@ version.workspace = true
 name = "nls"
 path = "src/main.rs"
 
-[features]
```

Comment:
I'm not sure we want to remove the `format` feature though. I think the rationale around features right now is to leave the possibility to build a minimal nickel interpreter that can be embedded say on a server, which doesn't need to format, render markdown, etc. - so anything that isn't strictly necessary and pulls in a non trivial set of dependencies is probably better kept behind a feature gate.

Reply:
This is just the nls format feature, though. Is there a need for a minimal nls?

Reply:
Ah, you're right, my bad.